build.gradle

```diff
@@ -2,25 +2,27 @@
 
 buildscript {
     repositories {
-        jcenter()
+        google()
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
-        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4.1'
-
+        classpath 'com.android.tools.build:gradle:7.0.4'
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
+plugins {
+    id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
+}
+
 allprojects {
     repositories {
-        jcenter()
         maven { url "https://jitpack.io" }
         maven {
             url 'https://maven.google.com'
         }
+        google()
```

Comment:
I think `google()` does the same as the `maven { url 'https://maven.google.com' }` above. Do we need both?

Reply:
I'll try out thank you :)